components/create/CreateLanding.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div
+    class="container is-flex is-align-items-center is-justify-content-center is-flex-direction-column">
+    <h1 class="title is-size-2">{{ $t('mint.landing.heading') }}</h1>
+
+    <div v-if="select === 'landing'" class="pack">
+      <nuxt-link class="pack-box" to="/create/collection">
+        <div class="pack-content">
+          <NeoIcon icon="image-polaroid" size="large" />
+          <div class="pack-content-text">
+            <p class="is-size-4 has-text-weight-bold">Collection</p>
+            <p>{{ $t('mint.landing.collection') }}</p>
+          </div>
+        </div>
+      </nuxt-link>
+      <a class="pack-box" @click="select = 'nft'">
+        <div class="pack-content">
+          <NeoIcon icon="gem" size="large" />
+          <div class="pack-content-text">
+            <p class="is-size-4 has-text-weight-bold">NFT</p>
+            <p>{{ $t('mint.landing.nft') }}</p>
+          </div>
+        </div>
+      </a>
+    </div>
+
+    <div v-if="select === 'nft'" class="pack">
+      <nuxt-link class="pack-box" to="/create/nft">
+        <div class="pack-content">
+          <NeoIcon icon="image" size="large" />
+          <div class="pack-content-text">
+            <p class="is-size-4 has-text-weight-bold">
+              {{ $t('mint.landing.singleNft') }}
+            </p>
+            <p>{{ $t('mint.landing.singleNftDesc') }}</p>
+          </div>
+        </div>
+      </nuxt-link>
+      <nuxt-link class="pack-box" :to="`/${urlPrefix}/massmint`">
+        <div class="pack-content">
+          <NeoIcon icon="photo-film" size="large" />
+          <div class="pack-content-text">
+            <p class="is-size-4 has-text-weight-bold">
+              {{ $t('mint.landing.massNft') }}
+            </p>
+            <p>{{ $t('mint.landing.massNftDesc') }}</p>
+          </div>
+        </div>
+      </nuxt-link>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoIcon } from '@kodadot1/brick'
+
+const { urlPrefix } = usePrefix()
+
+const select = ref<'landing' | 'nft'>('landing')
+</script>
+
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+
+.container {
```

Comment:
I would have create new layout if container is bad

Reply:
for now, no need to create a new layout I guess. create another one if we need another page something like this